apps/copyparty/overlays/nishir-tailnet/kustomization.yaml

```
@@ -20,4 +23,10 @@ patches:
       name: copyparty
       version: v1
       kind: NetworkPolicy
-  - path: patch-svc.yaml
+patches:
```

YAMLlint (1.37.1)

[error] duplication of key "patches" in mapping

(key-duplicates)